src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -640,9 +640,10 @@ class VM_Version : public Abstract_VM_Version {
   LP64_ONLY(static void clear_apx_test_state());
 
   static void clean_cpuFeatures()   { _features = 0; }
-  static void set_avx_cpuFeatures() { _features = (CPU_SSE | CPU_SSE2 | CPU_AVX | CPU_VZEROUPPER ); }
-  static void set_evex_cpuFeatures() { _features = (CPU_AVX512F | CPU_SSE | CPU_SSE2 | CPU_VZEROUPPER ); }
+  static void set_avx_cpuFeatures() { _features |= (CPU_SSE | CPU_SSE2 | CPU_AVX | CPU_VZEROUPPER ); }
+  static void set_evex_cpuFeatures() { _features |= (CPU_AVX512F | CPU_SSE | CPU_SSE2 | CPU_VZEROUPPER ); }
   static void set_apx_cpuFeatures() { _features |= CPU_APX_F; }
+  static void set_bmi1_cpuFeatures() { _features |= CPU_BMI1; }
```

Comment:
It would be better to change this as:
static void set_bmi_cpuFeatures() { _features |= (CPU_BMI1 | CPU_BMI2 | CPU_LZCNT); }
And then use set_bmi_cpuFeatures() in the gtest.

Reply:
Done.